src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -14006,22 +14006,20 @@ void Player_Action_8084F308(Player* this, PlayState* play) {
     }
 }
 
-void Player_Action_8084F390(Player* this, PlayState* play) {
-    CollisionPoly* floorPoly;
-    f32 sp50;
-    f32 sp4C;
-    f32 sp48;
-    s16 downwardSlopeYaw;
-    s16 sp44;
-    Vec3f slopeNormal;
-
+void Player_Action_SlipOnSlope(Player* this, PlayState* play) {
     this->stateFlags2 |= PLAYER_STATE2_5 | PLAYER_STATE2_6;
     LinkAnimation_Update(play, &this->skelAnime);
     func_8084269C(play, this);
     func_800F4138(&this->actor.projectedPos, NA_SE_PL_SLIP_LEVEL - SFX_FLAG, this->actor.speed);
 
     if (Player_ActionHandler_13(this, play) == 0) {
-        floorPoly = this->actor.floorPoly;
+        CollisionPoly* floorPoly = this->actor.floorPoly;
+        f32 horizontalSpeedTarget;
+        f32 horizontalSpeedIncrStep;
+        f32 horizontalSpeedDecrStep;
+        s16 downwardSlopeYaw;
+        s16 linkYawToStepTowards;
```

Comment:
I find this name a tad awkward, maybe `linkYawTarget` to match `horizontalSpeedTarget`

Reply:
just `shapeYawTarget` or simply `yawTarget` would be fine, and matches how other angle things in this file are named. no need to mention "link"

Reply:
Renamed it to `shapeYawTarget`.